README.md

```diff
@@ -1,14 +1,58 @@
 # rivet-add-on-boilerplate
-A starting point for developing Rivet add-ons
+A boilerplate for Rivet add-ons. Replace the contents of this README with information about your add-on, following the template in README.template.md
```

Comment:
Consider "A boilerplate for creating your own add-ons for the <a href="https://rivet.uits.iu.edu/">Rivet software design system</a>."